src/classes/emitter.ts

```diff
@@ -1,15 +1,18 @@
+// deno-lint-ignore-file no-explicit-any
```

Comment:
I don't like this. `isNotifyEvent` is only used once so I don't see the need for this helper!

Reply:
Ah indeed, I think I chose a bad example there. My point was mostly to make sure the `e` in the callback is strongly typed to something instead of `any` to make working with it in the future more transparent. But I agree, it's probably not worth the overhead as it is only one unique case..

Reply:
I totaly agree in https://github.com/surrealdb/surrealdb.deno/pull/4 for the deno repo (wich is replaced by this PR) I added better types for the events. As said before after this gets merged I will have a look and check what to optimize etc. 

But I didn't want to add it here now as this will hold this PR back for merging. (Same for formater below).

Reply:
I disabled the checks since I didn't want to add any more runtime changes since that isn't the goal of the pr (which already covers one too many scopes).

I'll be up for improving types/runtime behaviour in later prs :+1: